### PR TITLE
Transactional session v2

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: src/NServiceBus.Persistence.CosmosDB.TransactionalSession/NServiceBus.Persistence.CosmosDB.TransactionalSession.csproj
Comment (timbussmann) [inline src/NServiceBus.Persistence.CosmosDB.TransactionalSession/NServiceBus.Persistence.CosmosDB.TransactionalSession.csproj:15]: waiting for the rc package

**Comment by danielmarbach** [comment]
We need to first update to RC of the transactional session before we merge

**Review by timbussmann — APPROVED** [review]
approving already so it can be merged once the package dependencies to the tx session package have been updated

**Review by SzymonPobiega — APPROVED** [review]

**Pull request merged**
